neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.VM;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.SmartContract.Native
+{
+    public sealed class PolicyContract : NativeContract
+    {
+        public override string ServiceName => "Neo.Native.Policy";
+        public override ContractPropertyState Properties => ContractPropertyState.HasStorage;
+
+        private const byte Prefix_MaxTransactionsPerBlock = 23;
+        private const byte Prefix_MaxLowPriorityTransactionsPerBlock = 34;
+        private const byte Prefix_MaxLowPriorityTransactionSize = 29;
+        private const byte Prefix_FeePerByte = 10;
+        private const byte Prefix_BlockedAccounts = 15;
+
+        private bool CheckValidators(ApplicationEngine engine)
+        {
+            UInt256 prev_hash = engine.Snapshot.PersistingBlock.PrevHash;
+            TrimmedBlock prev_block = engine.Snapshot.Blocks[prev_hash];
+            return InteropService.CheckWitness(engine, prev_block.NextConsensus);
```

Comment:
How will be voted this signature?

Reply:
We can make a tool for validators to send the transactions.